packages/ui-components/src/components/windowedlist.ts

```diff
@@ -481,10 +486,10 @@ export abstract class WindowedListModel implements WindowedList.IModel {
    * @returns The viewport top position and its height
    */
   getSpan(startIndex: number, stopIndex: number): [number, number] {
-    const startSize = this._getItemMetadata(startIndex);
-    const top = startSize.offset;
+    const startSizer = this._getItemMetadata(startIndex);
```

Comment:
Why is there now `startSizer` (ending with `r`) and `stopSize` (no `r`)? I see `sizer` in the changes below. If `sizer` should be used from now on, should `stopSize` be `stopSizer`?

Reply:
Yes, it should all be size**r** because these are taken from `widgetSizers` array. I was annoyed with `startSize.size` as this is of course confusing (and why would "size" have an offset). But sizers are objects which obviously have a size and can have a position (and since we are in 1D space it is an offset).

I missed the `stopSize`, I will push a commit.

Reply:
Resolved in 2cfa77e.